Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n  <meta charset=\"UTF-8\">\r\n  <title>$Title$</title>\r\n</head>\r\n<body>\r\n$END$\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision 91338423267784dde4764274eb10c7c6b1f8c9f5)
+++ b/index.html	(date 1725888743931)
@@ -5,6 +5,6 @@
   <title>$Title$</title>
 </head>
 <body>
-$END$
+<script src=""></script>
 </body>
 </html>
\ No newline at end of file
Index: main.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.js b/main.js
--- a/main.js	(revision 91338423267784dde4764274eb10c7c6b1f8c9f5)
+++ b/main.js	(date 1725888702611)
@@ -1,0 +1,9 @@
+let age = 39;
+let name = "Viacheslav";
+let cobbler = true;
+console.log(age);
+console.log(name);
+console.log(cobbler);
+console.log(typeof(age));
+console.log(typeof(name));
+console.log(typeof(cobbler));
\ No newline at end of file
